Documentation/ContentObjects/Pageview/Index.rst

```diff
@@ -0,0 +1,435 @@
+:ref:`Extbase <t3coreapi:extbase>` plugins.
+
+..  contents:: Table of contents
+    :local:
+    :depth: 1
+
+..  _cobj-pageview-properties-example:
+
+Example: Display a page with Fluid templates
+============================================
+
+..  code-block:: typoscript
+    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+    page = PAGE
+    page {
+        10 = PAGEVIEW
+        10 {
+            paths {
+                100 = EXT:my_sitepackage/Resources/Private/Templates/
+            }
+            variables {
+                parentPageTitle = TEXT
+                parentPageTitle.data = levelfield:-1:title
+            }
+            dataProcessing {
+                10 = menu
+                10.as = mainMenu
+            }
+        }
+    }
+
+.. _cobj-pageview-data:
+
+Default variables in Fluid templates
+====================================
+
+..  contents::
+    :local:
+
+The following variables are available by default in the Fluid template.
+
+Additional variables can be defined with property
+:ref:`variables <cobj-pageview-variables>`.
+
+
+.. _cobj-pageview-data-language:
+
+language
+--------
+
+..  confval:: language
+    :name: pageview-data-language
+    :type: :php:`\TYPO3\CMS\Core\Site\Entity\SiteLanguage`
+
+    The current :php:`SiteLanguage` object, see also
+    :ref:`the SiteLanguage object <t3coreapi:sitehandling-sitelanguage-object>`.
+
+
+.. _cobj-pageview-data-language-example:
+
+Example: Display the site title in the current language
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+..  code-block:: html
+    :caption: EXT:my_sitepackage/Resources/Private/Templates/Pages/Default.html
+
+    <f:layout name="Default" />
+    <f:section name="Main">
+        <main role="main">
+            <p>The site title in the current template is: {language.websiteTitle}</p>
+        </main>
+    </f:section>
+
+.. _cobj-pageview-data-page:
+
+page
+----
+
+..  confval:: page
+    :name: pageview-data-page
+    :type: :php:`TYPO3\CMS\Frontend\Page\PageInformation`
+
+    The current `PageInformation` as object. See also
+    :ref:`Frontend page information <t3coreapi:typo3-request-attribute-frontend-page-information>`.
+
+.. _cobj-pageview-data-page-example:
+
+Example: Display the title and abstract of the current page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+..  code-block:: html
+    :caption: EXT:my_sitepackage/Resources/Private/Templates/Pages/Default.html
+
+    <f:layout name="Default" />
+    <f:section name="Main">
+        <main role="main">
+            <p>The title of the page with id {page.id} is: {page.pageRecord.title}. It has the
+                following abstract:</p>
+            <p>{page.pageRecord.abstract}</p>
+        </main>
+    </f:section>
+
+.. _cobj-pageview-data-settings:
+
+settings
+--------
+
+..  confval:: settings
+    :name: pageview-data-settings
+    :type: array
+
+    Contains all TypoScript settings (= Constants)
```

Comment:
Link to TypoScript Constants?

Reply:
I actually asked Stefan Bürk about this as I cannot really explain this variable yet

Reply:
I overhauled the section and added examples